content/docs/manage/integrations.md

```diff
@@ -26,3 +30,14 @@ To revoke an integration:
 1. Select a project.
 1. Select **Integrations**.
 1. Identify the integration you want to remove and click **Revoke**.
+
+## Supported integrations
+
+Neon currently supports the following integrations:
+
+- [Neon integration with Vercel](/docs/guides/vercel)
+- [Neon integration with Hasura](/docs/guides/hasura)
+
+<Admonition type="note">
+The **Integrations** page only shows Vercel integrations. Support for viewing other viewing other integrations will be added in a future release.
```

Comment:
duplicate "viewing".
btw we don't plan to add any other integrations yet

Reply:
Thanks, @duskpoet . Revised.